entries/e/easternbank.com.json

```diff
@@ -0,0 +1,15 @@
+{
+  "Eastern Bank": {
+    "domain": "easternbank.com",
+    "url": "https://www.easternbank.com/",
```

Comment:
```suggestion
```

The `url` key is only needed if going to the root domain (e.g. `https://easternbank.com`) fails

Reply:
* I removed the `url` key.
* I had already ran the PNG through TinyPNG, but @phallobst already (and graciously) gave me the SVG version of the logo.  (Thank you again!)
    * I am now using the SVG image.